docs/modules/kubernetes/pages/helm-upgrade-guide.adoc

```diff
@@ -0,0 +1,166 @@
+= Helm Upgrade Guide
+
+Before upgrading, review the Hazelcast xref:maintain-cluster:rolling-upgrades.adoc#hazelcast-members-compatibility-guarantees[].
+
+== Upgrade With Persistence Disabled
+
+`helm upgrade` command does not allow modifications to `spec.volumeClaimTemplates` property in stateful sets that prevents upgrade which can be seen in link:https://github.com/helm/charts/issues/7803[this helm issue]. Because of this issue, we have some upgrade limitations related to Management Center's persistence and hotRestart/persistence that uses volumeClaimTemplates internally. This guide informs you about possible upgrade scenarios.
+
+NOTE: If hotRestart (persistence) and the Management Center persistence feature are not enabled, all scenarios might not have been tested.
+
+We recommend to disable Management Center before upgrade, and re-enable it after upgrade. You should follow the steps outlined below:
+
+. Export the values from your existing installation:
++
+
+[source,bash]
+----
+helm get values <installation-name> > values.yaml
+----
+
+. Disable Management Center:
++
+
+[source,bash]
+----
+helm upgrade <installation-name> hazelcast/hazelcast-enterprise -f values.yaml --version <installed-version> --set mancenter.enabled=false
+----
+
+. Delete the persistent volume of the disabled Management Center:
++
+
+[source,bash]
+----
+kubectl get pvc -o name | grep "mancenter" | xargs -n 1 kubectl delete
+----
+
+. Upgrade your installation without disabling Management Center:
++
+
+[source,bash]
+----
+helm upgrade hz hazelcast/hazelcast-enterprise -f values.yaml --version <new-version>
+----
+
+== Upgrade with Persistence Enabled
+
+The following sections cover scenarios for upgrading with persistence/hotRestart enabled.
+
+=== Hazelcast Version 4.x to 5.x
+
+You can use rolling upgrades when moving from version 4.2 to version 5.0. For further information on rolling upgrades, see the xref:maintain-cluster:rolling-upgrades.adoc[] topic.
+
+NOTE: If upgrading from a Helm Chart of version 3.10 or earlier, rolling upgrades are not supported. For information on other ways to upgrade, see the xref:helm-upgrade-guide.adoc#alternative-solutions[] section below.
+
+If you are using version 3.11, we recommend upgrading to 5.3.9 as follows:
+
+[source,bash]
+----
+helm get values > values.yaml
+helm upgrade hz hazelcast/hazelcast-enterprise --version 5.3.9 -f values.yaml
+----
+
+== Hazelcast Version 5.x to 5.x
```

Comment:
I think this is part of the Upgrade with Persistence Enabled section, if so, the heading needs to be a level lower (=== Hazelcast instead of == Hazelcast)